opennebula/resource_opennebula_virtual_machine.go

```diff
@@ -560,6 +587,12 @@ func resourceOpennebulaVirtualMachineRead(d *schema.ResourceData, meta interface
 
 func flattenTemplate(d *schema.ResourceData, vmTemplate *vm.Template) error {
 
+	// VM Group
+	vmgmap := make([]map[string]interface{}, 0)
```

Comment:
to preallocate one element (before appending an element below) growing only slice capacity (not len): `make([]slice, 0, 1)`